velox/docs/functions/presto/regexp.rst

```diff
@@ -32,6 +32,12 @@ limited to 20 different expressions per instance and thread of execution.
         SELECT like('abc', '%b%'); -- true
         SELECT like('a_c', '%#_%', '#'); -- true
 
+    String sequence search: There are some patterns that are equivalent to simple
```

Comment:
This paragraph doesn't fit with the existing "Patterns 'hello',..." docs. It would be nice to make the whole doc consistent.

Reply:
updated with "hello velox"